plugins/spring-boot/spring-boot.service.spec.ts

```diff
@@ -28,19 +41,20 @@ describe("SpringBootService", () => {
       expect(tabs[2]).toEqual(traceTab);
     });
 
+
     it("should return two tabs", () => {
       // given
-      workspace.treeContainsDomainAndProperties.and.returnValues(true, false, false, false, true, true);
+      workspace.treeContainsDomainAndProperties.and.returnValues(true, false, true, false);
       // when
       const tabs = springBootService.getTabs();
       // then
       expect(tabs[0]).toEqual(healthTab);
-      expect(tabs[1]).toEqual(traceTab);
+      expect(tabs[1]).toEqual(loggersTab);
```

Comment:
why did you change this?  it changes the meaning of the test slightly. can't you make it pass with `traceTab`?